coalib/parsing/DefaultArgParser.py

```diff
@@ -105,6 +105,10 @@ def default_arg_parser(formatter_class=None):
     config_group = arg_parser.add_argument_group('Configuration')
 
     config_group.add_argument(
+        '-ex', '--exclude-sections', nargs='*', metavar='STR',
```

Comment:
Usually it is a one letter word with `-` because `-abc` is usually interpreted as `-a -b -c`. I suggest you use a one letter too...